docs/customization/repository.rst

```diff
@@ -67,29 +62,54 @@ As we are selecting Products we need to have a join to translations, because the
 We are sorting the results by the count of how many products are still available on hand, which is saved on the ``onHand`` field on the specific ``variant`` of each product.
 Then we are limiting the query to 8 by default, to get only 8 products that are low in stock.
 
-**2.** In order to use your repository you need to configure it in the ``config/_sylius.yaml``.
+**2.** In order to use your repository you need to configure it in the ``config/packages/_sylius.yaml``.
+As You see in ``_Sylius.yaml`` You already have basic configuration, now You just need add Yours repository and override resourceRepository
 
 .. code-block:: yaml
 
     sylius_product:
         resources:
             product:
                 classes:
-                    repository: App\Repository\ProductRepository
+                    model: App\Entity\Product\Product
+                    repository: App\Repository\ProductRepository # you need add yours repository to config file
+                translation:
+                    classes:
+                        model: App\Entity\Product\ProductTranslation
 
 **3.** After configuring the ``sylius.repository.product`` service has your ``findByOnHand()`` method available.
-You can form now on use your method in any **Controller**.
+You can form now on use your method in any **Controller**, or files which use ``sylius.repository.product``.
```

Comment:
Shouldn't it be `You can now use your method in any **Controller**, or files which use ``sylius.repository.product``.` ? Since the sentence above is just a modified version of a previously existing one, I'd summon @CoderMaggie here :D

Reply:
What kind of files are these? :D
Let's make it:
```suggestion
You can now use your method in anywhere when you are operating on the Product repository.
```
what do you think?